frontend/src/entities/dataset/create/dataset-create-functions.tsx

```diff
@@ -14,15 +14,18 @@
   limitations under the License.
 */
 
-import { IDataset } from '../../../shared/model/dataset.model';
+import { DatasetType, IDataset } from '../../../shared/model/dataset.model';
 import { determineScope } from '../dataset.service';
 
 export const createDatasetFromForm = (form: any, projectName: string, username: string): IDataset & Required<Pick<IDataset, 'name' | 'description' | 'type' | 'format' | 'scope'>> => {
+    if (form.type.startsWith(DatasetType.GROUP)) {
+        form.type = DatasetType.GROUP;
+    }
```

Comment:
<jackie-chan-why /> ?

Reply:
I don't even remember writing this... I don't see the point of it either, will remove it